localdev/config/armada/config.yaml

```diff
@@ -25,8 +38,9 @@ eventapi:
   postgres:
     connection:
       host: "postgres"
+defaultToLegacyEvents: true
 pulsar:
-  enabled: true
+  enabled: false
```

Comment:
Should pulsar be getting disabled?

Reply:
armadactl watch does not seem to work with this set true. now matches the config in e2e

Reply:
Interesting. I'll see if this changes anything for me.